test/units/testsuite-17.credentials.sh

```diff
@@ -4,6 +4,11 @@
 set -eux
 set -o pipefail
 
+if ! systemctl is-enabled systemd-udev-load-credentials.service; then
+    echo no systemd-udev-load-credentials.service >/skipped
```

Comment:
I know it's not necessary to quote strings for `echo`, but it makes code highlighting work properly.

```suggestion
    echo "Misssing systemd-udev-load-credentials.service" >>/skipped
```